chromium_src/net/tools/transport_security_state_generator/input_file_parsers.cc

```diff
@@ -691,25 +690,25 @@ HMUfpIBvFSDJ3gyICh3WZlXi/EjJKSZp4A==
 
 GlobalSignRootCA_R3
```

Comment:
@fmarier yes that's the same. is it actually the same cert but with the line length decreased by 1? for instance the first line in each is:

MIIDXzCCAkegAwIBAgILBAAAAAABIVhTCKIwDQYJKoZIhvcNAQELBQAwTDEgMB4G
MIIDXzCCAkegAwIBAgILBAAAAAABIVhTCKIwDQYJKoZIhvcNAQELBQAwTDEgMB4

Reply:
You're right, I didn't even think of that.

I've just removed all newlines and compared the two. They are identical.

Reply:
I'll keep that change in since we may be wondering again later why our file doesn't match the one on the website, but I will update the commit message.